### PR TITLE
Breaking change: Uses specific value for db connection string.

### PR DESCRIPTION
**Pull request by abergs**
Two motivations:
* Simplify to only have one config value to decide connection string
* Bug: We didn't properly use our new values

### CODE REVIEW

**Comment by abergs** [comment]
The need to merge this change is if we think it will be valuable to have different name or if we should let the execution env figure out envrionment variabler conflits, like in: https://github.com/passwordless/passwordless-server/commit/bb5b6e86c36effe039106529d15e3bfeee565b64

File: src/Api/Helpers/AddDatabaseExtensionMethod.cs
Comment (justindbaur) [inline src/Api/Helpers/AddDatabaseExtensionMethod.cs:28]: It's fine to remove this but I'll explain why I did this and leave the decision up to you. By using `IConfiguration` from the service provider instead of the outer scope one you are able to keep this method `static`. This way it won't capture any outer scope, requires less allocations, and should run a tad faster.

There are no bugs that should arise from capturing `IConfiguration` because it's a singleton, it's lifetime will match that of the Host so it won't unnecessarily be kept from being GC'ed (because it won't ever be), and changes to it aren't done through creating a new reference. So in this instance it's safe but I generally try to not capture scope if I can help it. 

Decision is up to you though.

File: src/Api/appsettings.Development.json
Comment (justindbaur) [inline src/Api/appsettings.Development.json:3]: Same thing.
```suggestion
    "sqlite": {
      "api": "Data Source=passwordless_dev.db"
     }
```

File: src/AdminConsole/appsettings.Development.json
Comment (justindbaur) [inline src/AdminConsole/appsettings.Development.json:14]: In JSON I vote to represent it with object syntax.
```suggestion
    "sqlite": {
      "admin": "Data Source=adminconsole_dev.db"
    }
```

**Review by justindbaur — APPROVED** [review]
Some suggestions and comments but overall a good improvement that has my seal of approval.

File: src/Api/Helpers/AddDatabaseExtensionMethod.cs
Comment (abergs) [reply src/Api/Helpers/AddDatabaseExtensionMethod.cs:28]: Aha I see, yeah that's a valid point. Will add it back with a explainer.

**Pull request merged**
